.gitignore

```diff
@@ -4,6 +4,7 @@
 example.tf
 terraform.tfplan
 terraform.tfstate
+secret.tfvars
```

Comment:
Intentional?

Reply:
Yep, I made this change last time I was testing something and didn't get around to committing it, but it's worth having in main.